fenix/app/src/main/res/xml/search_preferences.xml

```diff
@@ -62,5 +62,13 @@
             android:layout="@layout/checkbox_left_sub_preference"
             android:title="@string/preferences_show_search_suggestions_in_private"
             app:iconSpaceReserved="false" />
+        <SwitchPreference
+            android:defaultValue="false"
+            android:key="@string/pref_key_show_nonsponsored_suggestions"
+            android:title="@string/preferences_show_nonsponsored_suggestions" />
+        <SwitchPreference
+            android:defaultValue="false"
+            android:key="@string/pref_key_show_sponsored_suggestions"
+            android:title="@string/preferences_show_sponsored_suggestions" />
```

Comment:
I'm wondering, is `search_preferences` still used?

Reply:
Oh, I see, this looks like it's the "old", non-unified search UI? I asked on Slack, with a bit more context! 😁